packages/honeycomb-opentelemetry-web/src/console-trace-link-exporter.ts

```diff
@@ -32,10 +37,16 @@ export function configureConsoleTraceLinkExporter(
  */
 class ConsoleTraceLinkExporter implements SpanExporter {
   private _traceUrl = '';
+  private _logLevel: DiagLogLevel = DiagLogLevel.DEBUG;
+
+  constructor(serviceName?: string, apikey?: string, logLevel?: DiagLogLevel) {
+    if (logLevel) {
+      this._logLevel = logLevel;
+    }
 
-  constructor(serviceName?: string, apikey?: string) {
     if (!serviceName || !apikey) {
-      console.debug(MISSING_FIELDS_FOR_LOCAL_VISUALIZATIONS);
+      this._logLevel >= DiagLogLevel.DEBUG &&
```

Comment:
Let's move this into a nested if clause so that it's explicit. Fall through side effects can be difficult to reason over.